rendering/markup_render_test.go

```diff
@@ -34,6 +34,17 @@ func TestRenderMarkdownContentWithFenceHighlighter(t *testing.T) {
 	assert.True(t, strings.Contains(result, "<span class=\"kwd\">func</span>"))
 }
 
+func TestRenderMarkdownContentWithCheckboxItems(t *testing.T) {
+	content := "* [] Some Item 0\n* [ ] Some Item 1\n* [X] Some Item 2\n* [x] Some Item 3"
```

Comment:
Please also add a sub-test for this dash-based kind of rendering instead of the star-based list. Oh, and note, that you can use backticks to have nicer strings and line breaks.

```go
content := `
- [ ] foo
- [b] bar
`
```

Reply:
I can add that test, but the -/* handling is not done in our code at all. This is done in the upstream markdown implementation.

Reply:
> Done.

@michaelkleinhenz where? ;)

Reply:
In the test.